kentikapi/internal/api_payloads/device.go

```diff
@@ -273,7 +273,7 @@ func privacyProtocolFromStringPtr(s *string) *models.PrivacyProtocol {
 	return &result
 }
 
-func snmp3ConfToPayload(d *models.SNMPv3Conf) *snmpv3ConfPayload {
+func snmp3ConfToPayload(d *models.SNMPv3Conf) *snmpV3ConfPayload {
```

Comment:
This should be `snmpV3ConfToPayload` for consistency (since you are fixing details).

Reply:
Done